script.js

```diff
@@ -0,0 +1,746 @@
+"use strict";
+
+// ======= constants =======
+var TITLE = "WABA Bike Infrastructure Map Project";
```

Comment:
Is this variable being used? I don't see it elsewhere in the code. If it's not being used, then remove. If it is being used, then it should be camelcase `var title` instead of capitalized.